go.mod

```diff
@@ -44,6 +44,8 @@ require (
 	gorm.io/plugin/dbresolver v1.4.7
 )
 
+require github.com/teivah/onecontext v1.3.0 // indirect
+
```

Comment:
Q: This package lacks maintenance and has less Stars, can we use self-implementation?